src/DwitTech.AccountService.Core/Services/SecurityService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using DwitTech.AccountService.Core.Interfaces;
+using System.Security.Cryptography;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Core.Services
+{
+    public class SecurityService
+    {
+        private string ConvertByteToString(byte[] hashpassword)
+        {
+            StringBuilder hashedstringoutput = new StringBuilder(hashpassword.Length);
```

Comment:
variable and parameter names should be camelCase. All your variables and parameters are guilty of this.